reference-implementation/README.md

```diff
@@ -12,30 +12,47 @@ Test coverage is not complete, but we do aim for it to be. Adding tests would be
 
 You can check the test coverage at any time by running `npm run coverage` in this folder.
 
-### Original Tests
+To run all tests (and the lint step), run `npm test` in this folder.
 
-The original tests are written using the [tape](https://github.com/substack/tape) framework.
+### Legacy tests
 
-- To run them, type `npm test` in this folder.
-- To run a specific test file, type e.g. `node test-runner test/readable-stream.js`.
+The original tests are written using the [tape](https://github.com/substack/tape) framework. They live in the `test` subfolder of this folder.
+
+- To run them, type `npm run legacy-test` in this folder.
+- To run a specific test file, type e.g. `node run-tests.js test/readable-stream.js`.
```

Comment:
IMO once we get it down to just abstract-ops.js we should eliminate the tests as maintaining the separate framework is a burden. But yeah, I'll change it for now.